process/physics.py

```diff
@@ -4347,7 +4347,9 @@ def igmarcal(self):
             f"{'':>34}{'for H = 1':<20}power balance",
         )
 
-        for iisc in range(32, 48):
+        # for iisc in range(32, 48):
+        # Put the ITPA value first
+        for iisc in [50, 34, 37, 38, 39, 46, 47, 48]:
```

Comment:
Why the sudden change in the scaling laws checked?

Reply:
Because no-one will ever look at all those quirky scaling laws.  I only left the ST and stellarator ones in because I don't know anything about those models and didn't want to rock the boat.